### PR TITLE
Fix eslint errors in AssetManager.js, cache-manager.js

### PR DESCRIPTION
**Pull request by dumganhar**
Re: #

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Comment by dumganhar** [comment]
@cocos-robot run test cases

**Comment by dumganhar** [comment]
@cocos-robot run test cases

**Comment by dumganhar** [comment]
@cocos-robot run test cases

**Comment by dumganhar** [comment]
@cocos-robot run test cases

File: platforms/minigame/platforms/alipay/wrapper/fs-utils.js [hunk 1]
```diff
@@ -22,6 +22,7 @@
  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  THE SOFTWARE.
  ****************************************************************************/
+/* eslint-disable no-undef */
```
Comment (minggo) [inline platforms/minigame/platforms/alipay/wrapper/fs-utils.js:25]: Why need it?

Comment (dumganhar) [reply platforms/minigame/platforms/alipay/wrapper/fs-utils.js:25]: The global variable `my` is defined by minigame platform which couldn't be recognized.
 The eslint error is : 
```
'my' is not defined.eslint[no-undef](https://eslint.org/docs/latest/rules/no-undef)
```

**Comment by dumganhar** [comment]
@cocos-robot run test cases

**Review by minggo — APPROVED** [review]

**Pull request merged**